### PR TITLE
add npm + semistandard badges to README (fixes #703)

### LINKED ISSUE #703: Add travis, npm badges to README
à la https://raw.githubusercontent.com/mattdesl/budo/master/README.md

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by cvan** [comment]
can preview here: https://github.com/aframevr/aframe-core/blob/badges/README.md

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-
-> There are currently plans for this repository to be the canonical [`aframe`](https://github.com/aframevr/aframe).
+[![build status][travis-image]][travis-url]
+[![NPM version][npm-image]][npm-url]
+[![Downloads][downloads-image]][downloads-url]
+[![js-semistandard-style][semistandard-image]][semistandard-url]
 
 A-Frame is a library for creating virtual reality web experiences.
 
+> There are currently plans for this repository to be the canonical [`aframe`](https://github.com/aframevr/aframe).
```
Comment (ngokevin) [inline README.md]: nitpick but can we move this above the tagline? i feel it interrupts the introduction.

**Comment by ngokevin** [comment]
r+wc, you can press merge once it's updated

**Comment by ngokevin** [comment]
nice

File: README.md (hunk 1 above)
Comment (cvan) [inline README.md]: I thought it was a weird thing to put _above_ the description, but sure

**Pull request merged**